frontend/src/libs/components/pagination/pagination.tsx

```diff
@@ -0,0 +1,152 @@
+import { getValidClassNames } from '~/libs/helpers/helpers.js';
+import { useCallback } from '~/libs/hooks/hooks.js';
+
+import { Button } from '../button/button.jsx';
+import { Dropdown } from '../components.js';
```

Comment:
Be careful with that, using import from `components.js` and then re-exporting Pagination in the same file is a circular dependency

Reply:
Auto import always uses import from the component. I forgot to check it